src/pages/messages/components/TicketList.tsx

```diff
@@ -73,6 +70,17 @@ export const TicketList = ({ ticketsQuery, setRowsPerPage, rowsPerPage, setPage,
 
   const searchParams = new URLSearchParams(location.search);
   const viewedByNotParam = searchParams.get(TicketSearchParam.ViewedByNot) || 'show-all';
+  const resultsParam = searchParams.get(TicketSearchParam.Results);
+  const fromParam = searchParams.get(TicketSearchParam.From);
+  const rowsPerPage = (resultsParam && +resultsParam) || ROWS_PER_PAGE_OPTIONS[0];
+  const page = (fromParam && resultsParam && Math.floor(+fromParam / rowsPerPage)) || 0;
+
+  const updatePath = (from: string, results: string) => {
+    const syncedParams = syncParamsWithSearchFields(searchParams);
+    syncedParams.set(TicketSearchParam.From, from);
+    syncedParams.set(TicketSearchParam.Results, results);
+    navigate({ search: syncedParams.toString() });
+  };
```

Comment:
Mulig dette kunne vært en egen hook?

Reply:
Hvordan da tenker du?

En annen ting som kunne vært hook er henting av alle params, som vi feks har for NVI i `useNviCandidatesParams.ts`. Det fjerner mye støy for henting av URL params. 
Tror noen en av de nye hookene i react-router kan hjelpe oss mye med håndtering av URL params også da